### PR TITLE
[M1160] add root-level project properties to PSSEs

### PR DESCRIPTION
**Pull request by saanobhaai**
## Summary by CodeRabbit

- **New Features**
  - Introduced new fields in project summary models to capture project administrators and data policies.
  - Added filtering capabilities for project summaries based on project ID, name, and administrators.

- **Bug Fixes**
  - Enhanced the project summary update process to include new fields and improve data handling.

- **Documentation**
  - Updated documentation to reflect the new fields and filtering options available for project summaries.

### CODE REVIEW

**Comment by saanobhaai** [comment]
[display project properties in single-project metrics pane for projects with no SEs](https://trello.com/c/sdURFmIe/1160-display-project-properties-in-single-project-metrics-pane-for-projects-with-no-ses)

**Comment by saanobhaai** [comment]
@coderabbitai review

File: src/api/migrations/0073_auto_20241114_1443.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        migrations.AddField(
+            model_name="restrictedprojectsummarysampleevent",
+            name="data_policy_bleachingqc",
+            field=models.CharField(default="awaiting refresh", max_length=50),
+        ),
+        migrations.AddField(
+            model_name="restrictedprojectsummarysampleevent",
+            name="data_policy_habitatcomplexity",
+            field=models.CharField(default="awaiting refresh", max_length=50),
+        ),
```
[the remark this answers is not in the document]
Comment (saanobhaai) [reply src/api/migrations/0073_auto_20241114_1443.py:51]: I'm already doing that.

File: src/api/utils/summary_cache.py [hunk 2]
```diff
@@ -202,9 +203,21 @@ def _update_project_summary_sample_events(
...
+        admins = project.profiles.filter(role=ProjectProfile.ADMIN)
+        project_admins = [{"id": str(pa.pk), "name": pa.profile_name} for pa in admins]
+        data_policies = dict(Project.DATA_POLICIES)
+
         proj_summary_se_model.objects.create(
             project_id=project_id,
             project_name=project.name,
+            project_admins=project_admins,
+            project_notes=project.notes,
+            data_policy_beltfish=data_policies.get(project.data_policy_beltfish),
```
Comment (gridcell) [inline src/api/utils/summary_cache.py:215]: if they don't exist should they either default to strict or what we currently have set as default?

**Review by gridcell — APPROVED** [review]

**Pull request merged**